parser-typechecker/src/Unison/TermParser.hs

```diff
@@ -21,7 +21,8 @@ import           Prelude hiding (and, or, seq)
 import qualified Text.Megaparsec as P
 import qualified Unison.ABT as ABT
 import qualified Unison.Lexer as L
-import           Unison.Parser
+import           Unison.Parser hiding (seq)
+import qualified Unison.Parser as Parser (seq)
```

Comment:
Since there's another function named `seq` in this file, this makes it so this one has to be referenced as `Parser.seq` [here](https://github.com/unisonweb/unison/pull/467/files#diff-211d62c2ee7dccd868dc459a4e36136cR221).

Reply:
Ah roger.  I think you can leave the (seq) off the second one and still reference it as `Parser.seq`. nbd though.